src/plugin.js

```diff
@@ -530,6 +544,22 @@ class VR extends Plugin {
       this.triggerError_({code: 'web-vr-not-supported', dismiss: false});
     }
 
+    if (this.options_.enableOmnitone) {
```

Comment:
instead of `enableOmnitone`, we can have an `Omnitone` property that is a reference to the lib with current `options_.omnitone` prop renamed to `omnitoneOptions`.